doc/mp-test.md

```diff
@@ -46,27 +47,33 @@ In addition, when using `collect` and `long-run` there are additional optional a
 
 `mp-test collect`
 ```
-usage: mp_test.py collect [-h] [-n TIMES] [-w WAIT]
+usage: mp-test collect [-h] [-n [0-999]] [-w COLLECTION_WINDOW_START] [-t TIMEOUT]
 
-optional arguments:
+options:
   -h, --help            show this help message and exit
-  -n TIMES, --times TIMES
-                        Run the given method 'n' times.
-  -w WAIT, --wait WAIT  Amount of time to wait between collections (in seconds).
+  -n [0-999], --collection-number [0-999]
+                        Start at a custom collection number instead of 0.
+  -w COLLECTION_WINDOW_START, --collection-window-start COLLECTION_WINDOW_START
+                        Sets a custom collection window start time to time before present. The collection window end time will always be the
```

Comment:
what is the format of this argument? time since epoch in milliseconds?

Reply:
Does our server already pass a collection window start to the adapter? If so, what does `mp-test` send now?

Reply:
Time before present, using the standard way we've been parsing time, e.g., `20m` => 20 minutes before present; `30s` => 30 seconds before present, etc. I'll see if I can make it a bit clearer

Reply:
mp-test passes nothing now. The server keeps track of the collection number and window. This doesn't change that, it just adds a way for us to override what the server passes to the adapter.

Reply:
For example, if you start the `wait` command, and the initiate a 'collect' request passing in "collectionNumber": 20, then the server will use that number instead of its own ('0'). If you then send another collect request but omit the "collectionNumber" property, it will revert to what the server thinks is correct and pass '1' to the adapter (since this is the second collection from its point of view)